Common/Brokerages/BrokerageName.cs

```diff
@@ -59,7 +59,11 @@ public enum BrokerageName
         /// <summary>
         /// Transaction and submit/execution rules will use gdax models
         /// </summary>
-        GDAX = 12
+        GDAX = 12,
```

Comment:
Hmm, just noticing that this is set to 12 explicitly -- @StefanoRaggi -- do you know of any particular reason why this is hard coded here instead of using the auto-incrementing enum values?